notion/README.md

```diff
@@ -0,0 +1,20 @@
+# Agent Check: Notion
+
+## Overview
+
+Event monitoring for Notion
```

Comment:
The overview section should include the following: 

1. What does Notion do in 1-2 sentences
2. What will customers get from this integration? This should be clear to mutual customers and potential new customers 
3. What types of data are sent in and the value?